### ISSUE TITLE
const enums results in false positive with the rule "no-shadow"

### ISSUE THREAD

**Issue by onderceylan** [typescript-eslint/tslint-to-eslint-config#856]
### 🐛 Bug Report

Thanks for this great migration library, it saves a lot of time!
I've had an issue with my migration due to default eslint rule "no-shadow". It should be replaced with "@typescript-eslint/no-shadow".

-   `tslint-to-eslint-config` version: 2.0.1
-   ESLint version: 7.6.0
-   Node version: 12.16.1

#### Actual Behavior

```
"no-shadow": ["error"]
```

#### Expected Behavior

```
"no-shadow": "off",
"@typescript-eslint/no-shadow": ["error"]
```

#### Reproduction

```
export const enum TestingConstEnumShadowVar {
  First,
  Second,
  Third,
}
```

**Comment by arizonatribe** [comment]
Did you even fix this issue or is your "fix" to have people disable an ESLint rule (which isn't a fix)?

**Comment by JoshuaKGoldberg** [comment]
@arizonatribe https://github.com/typescript-eslint/tslint-to-eslint-config/pull/860/files

**Comment by arizonatribe** [comment]
I ask because I configured the `overrides` section of a local eslint config (for TS files only) to shut off ESLint's "no-shadow" and to enable the corresponding rule for this package, but when I tested it by intentionally creating a local function that declared a scoped variable which shadows another in the same file, there weren't any linting warnings.

**Comment by JoshuaKGoldberg** [comment]
Interesting! @arizonatribe could you please file a new issue following the issue template? Most of the time these reports are from an outdated typescript-eslint package, so we'll want to check for that.

**Comment by arizonatribe** [comment]
Sure, but these are the versions of the relevant packages we're using in our project:

@typescript-eslint/parser: 4.15.0
@typescript-eslint/eslint-plugin: 4.15.0
eslint: 6.8.0

**Comment by arizonatribe** [comment]
You know, this was my bad; sorry was a long day of TypeScript refactoring and a lot of frustration from the many official Microsoft github issue threads (for TypeScript and VSCode) where they tended to get dismissive and would lock threads without solving the issue (Brought back PTSD from years of being stuck in .NET shops).

The issue I was running into here was caused because I had this in my eslint config:

```
  parser: "@typescript-eslint/parser",
  plugins: ["@typescript-eslint"],
  extends: ["airbnb-base"],
  plugins: ["import", "jsdoc"],
```

instead of this:
```
  parser: "@typescript-eslint/parser",
  plugins: ["@typescript-eslint", "import", "jsdoc"],
  extends: ["airbnb-base"],
```

Obviously I was overwriting the `@typescript-eslint` configuration` with the second `plugins`. Thanks for your response and patience!

**Comment by JoshuaKGoldberg** [comment]
Aha! Glad it worked out in the end. And congrats / condolences for the long day of GitHub issue spelunking, that sounds painful.

**Comment by exapsy** [comment]
For me it was `jsdoc` missing from `plugins`. Im able to reproduce the issue when `jsdoc` is missing

```json
  plugins: [
    '@typescript-eslint',
    'import',
    'eslint-plugin-import',
    'eslint-plugin-react',
    'eslint-plugin-unicorn',
    'eslint-plugin-prefer-arrow',
    'i18next',
    'react-hooks',
    'jsdoc' // Remove this to reproduce issue
  ],
```
